README.md

```diff
@@ -21,10 +21,17 @@ NuPIC binaries are available for:
 - [wheel](http://pythonwheels.com)
 - [numpy](http://www.numpy.org/)
 
-### Install
+### Install OS X
 
     pip install nupic
 
+### Install Linux
+
+    pip install https://s3-us-west-2.amazonaws.com/artifacts.numenta.org/numenta/nupic.core/releases/nupic.bindings/nupic.bindings-0.1.5-cp27-none-linux_x86_64.whl
+    pip install nupic
+
+> **NOTE**: The `nupic.bindings` binary distribution is not stored on [Pypi](https://pypi.python.org/pypi/nupic) along with the OS X distribution. NuPIC uses the [wheel](http://pythonwheels.com) binary format, and Pypi does not support hosting Linux wheel files. So we are forced to host our own.
```

Comment:
Move this note before the commands so people understand why there are wacky installation commands.